sections/STA.php

```diff
@@ -2,6 +2,8 @@
     <div class="container text-center">
         <h2 class="mb-4" style="font-size: 2.5rem; font-weight: bold;">Хотите узнать больше?</h2>
         <p class="mb-4" style="font-size: 1.2rem;">Запишитесь на бесплатную консультацию с нашими экспертами</p>
-        <a href="#contact-form" class="btn btn-lg btn-outline-dark" style="padding: 15px 30px; border-radius: 50px;">Записаться сейчас</a>
+        <button class="btn btn-lg btn-outline-dark" style="padding: 15px 30px; border-radius: 50px;" data-bs-toggle="modal" data-bs-target="#askQuestionModal">
```

Comment:
please, remove style and use bootstrap class

for padding:
https://getbootstrap.com/docs/5.0/utilities/spacing/#margin-and-padding

for border-radius:
https://getbootstrap.com/docs/5.0/utilities/borders/#sizes

Reply:
done